modules/Quiz/resources/quiz.js

```diff
@@ -790,9 +790,25 @@
             if($.cpObject.cpArray[selectedQuestion].questionType === this.KIVQModule.QUESTIONS_TYPE.OPEN_QUESTION 
                 && $.cpObject.cpArray[selectedQuestion].openAnswer){
                 var text = $.cpObject.cpArray[selectedQuestion].openAnswer;
+                var feedback = $.cpObject.cpArray[selectedQuestion].feedback;
                 text = text.replace(/\n/g,"<br/>");
                 $(".yourAnswer").html(text).addClass("open-question-answer");
                 $(".correctAnswerText").html("");
+                if (feedback) {
+                    console.log($.cpObject)
+                    $(".feedback").html("Feedback").bind('click', function () {
+                        $(".feedback-modal").show();
+                        $(".reviewAnswerPlace").hide();
+                        $(".reviewAnswerFooter").hide();
+
+                    });
+                    $(".feedback-modal .icon-close").bind('click', function () {
```

Comment:
Accessibility features: 

1 make sure that after pop-up we need to set the focus to the X button (JQ focus) 
tab loops (might need manual management) between close and text 
need to be able to open the feedback with KB 
Need to be able to close the pop-up using tab and KB only (space OR enter)
Make sure that the professor feedback is accessible to screen-readers.